name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/TradeRepublicPDFExtractor.java

```diff
@@ -1037,7 +1040,124 @@ private void addDeliveryInOutBoundTransaction()
         addTaxesSectionsTransaction(pdfTransaction2, type);
         addFeesSectionsTransaction(pdfTransaction2, type);
     }
+    
+    private void addTaxInOutBoundTransaction()
+    {
+        DocumentType type = new DocumentType("(STEUERLICHER UMTAUSCH)", (context, lines) -> {
+            Pattern pDate = Pattern.compile("^(.*) DATUM (?<date>[\\d]{2}\\.[\\d]{2}\\.[\\d]{4})$");
+            context.put("transactionPosition", "0");
+
+            for (String line : lines)
+            {
+                Matcher mDate = pDate.matcher(line);
+                if (mDate.matches())
+                    context.put("date", mDate.group("date"));
+
+            }
+        });
+        this.addDocumentTyp(type);
```

Comment:
You could extract the date from the document in this way.
It should also be possible to transfer the security data in this way.
```suggestion
        final DocumentType type = new DocumentType("STEUERLICHER UMTAUSCH", //
                        documentContext -> documentContext //
                                        // @formatter:off
                                        // Straße 1 DATUM 07.12.2023
                                        // @formatter:on
                                        .section("date") //
                                        .match("^(.*) DATUM (?<date>[\\d]{2}\\.[\\d]{2}\\.[\\d]{4})$") //
                                        .assign((ctx, v) -> ctx.put("date", asDate(v.get("date")))));

        this.addDocumentTyp(type);
```